toolchain/check/handle_function.cpp

```diff
@@ -209,18 +209,19 @@ static auto BuildFunctionDecl(Context& context,
           .Case(KeywordModifierSet::Impl,
                 SemIR::Function::VirtualModifier::Impl)
           .Default(SemIR::Function::VirtualModifier::None);
+  SemIR::Class* class_info = nullptr;
```

Comment:
What do you think about renaming this to `virtual_class_info` or `vptr_class_info` or something, given that we only set it if this function is virtual, and below we assume that it being set means this function is virtual.

Reply:
Yeah, that's fair, done in ae421105c174f0e05f21e693c3fb2c44293bb9b8 (& removed the extra `is_dynamic` check below, since it's only non-null when it is dynamic anyway)